README.md

```diff
@@ -1,31 +1,39 @@
-# Exercism Canonical Data Syncer
+# Configlet
```

Comment:
Good idea. Will do.

Reply:
Actually, this change is potentially confusing, because the version string of "v2 configlet" starts with `v3` (the latest release is `v3.9.2` / `v3.9.10`).

Perhaps this?

```diff
- # Configlet v3
+ # Configlet for Exercism v3 
```

Reply:
Sure!

Reply:
Done.